src/shmessy/validators/unix_timestamp.py

```diff
@@ -50,7 +50,7 @@ def validate(self, data: ndarray) -> Optional[InferredField]:
             return None
 
         try:
-            selected_resolution = self._unix_timestamp_resolution(float(data[1]))
+            selected_resolution = self._unix_timestamp_resolution(float(data[0]))
```

Comment:
if a user makes the mistake I made this week this will still not work

Reply:
@michaelpecan can you please share your file that causes this issue?